www/app/visits/menu/menu.controller.js

```diff
@@ -0,0 +1,12 @@
+(function() {
```

Comment:
this file needs line changes
you can see https://github.com/kescobar-pernix/stein-frontend/pull/2/files